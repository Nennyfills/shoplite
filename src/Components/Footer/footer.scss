.footer {
  background-color: #fcfcfa;
  position: absolute;
  bottom: 0;
  width: 100%;
  padding: 20px 0;
  box-shadow: 0 -5px 3px -3px #ccc, 0 5px 3px -3px #ccc;
  text-align: center;
}
