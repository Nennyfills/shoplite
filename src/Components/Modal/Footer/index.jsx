import React from 'react';
import PropTypes from 'prop-types';

import Button from 'Components/Button';
import '../modal.scss';

const ModalFooter = ({ currencies, total, onClick }) => {
  return (
    <div className="modal-footer" data-test="modal-footer">
      <div></div>
      <section className="top">
        <h4>Subtotal</h4>
        <side>{`${currencies} ${total}`}</side>
      </section>
      <section className="bottom">
        <Button name="Proceed To CheckOut" onClick={onClick} />
      </section>
    </div>
  );
};

ModalFooter.defaultProps = {
  total: 0,
  currencies: 'NGN',
  onClick: () => '',
};

ModalFooter.propTypes = {
  total: PropTypes.number,
  currencies: PropTypes.string,
  onClick: PropTypes.func,
};

export default ModalFooter;
