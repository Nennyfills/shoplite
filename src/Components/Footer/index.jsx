import React from 'react';

import './footer.scss';

const Footer = () => {
  const date = new Date();
  const getYear = date.getFullYear() + '  ';
  return (
    <section className="footer" data-test="footer">
      Lumin &copy; <span>{getYear}</span>
    </section>
  );
};

export default Footer;
