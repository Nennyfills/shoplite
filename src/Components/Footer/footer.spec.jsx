import React from 'react';
import { shallow } from 'enzyme';

import Footer from '.';

const setUp = (props = {}) => {
  const component = shallow(<Footer />);
  return component;
};

describe('Footer', () => {
  let component;
  beforeEach(() => {
    component = setUp({});
  });
  it('Should render without errors', () => {
    expect(component).toMatchSnapshot();
  });
  it('Should render footer section', () => {
    const wrapper = component.find(`[data-test='footer']`);
    expect(wrapper.length).toBe(1);
  });
});
