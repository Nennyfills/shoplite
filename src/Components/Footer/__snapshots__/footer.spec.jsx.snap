// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render without errors 1`] = `ShallowWrapper {}`;
